### DISCUSSION TITLE
parameters Argo CD sends to helm template

### DISCUSSION THREAD

**Discussion by ira-gordin-sap** [argoproj/argo-cd#7797] [Q&A]
I am changing my Argo CD application to use argocd-vault-plugin-helm (https://ibm.github.io/argocd-vault-plugin/v1.5.0/usage/#with-helm) as source instead of using helm as source in order to read secrets from Vault. I am searching for full list of parameters Argo CD sends to helm template in order to align the plugin with it.
I suppose "--namespace" is one of them.

**Comment by chetan-rns** [comment]
Hi @ira-gordin-sap, helm parameters can be specified as shown here https://argo-cd.readthedocs.io/en/stable/user-guide/helm/#helm-parameters. These parameters will be passed on to the helm template command.

**Comment by ira-gordin-sap** [comment]
Hi @chetan-rns, the issue is that Argo CD doesn't support more then one source. My application uses helm as a source and the only parameters are valueFiles. I should move to use plugin as source in order to use argocd-vault-plugin-helm (https://ibm.github.io/argocd-vault-plugin/v1.5.0/usage/#with-helm). 
I've that this is not enough define only valueFiles in this case. I've found empirically that I should add ARGOCD_APP_NAME and 
ARGOCD_APP_NAMESPACE and I am trying to understand whether I should sent more parameters in order my applications will work properly.
